sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -13,6 +18,7 @@
     private final Integer colno;
     private final String absPath;
     private final String platform;
+    private final Map<String, Object> vars;
```

Comment:
Oh, I guess if that's what it's called in the payload it probably makes sense to keep it consistent here then (even if it is not as specific as it could be.)